test-data/unit/check-namedtuple.test

```diff
@@ -185,3 +185,35 @@ class C(B): pass
 B(1).b
 C(2).b
 [builtins fixtures/property.py]
+
+[case testNamedTupleAsDict]
+from collections import namedtuple
+
+X = namedtuple('X', ['x', 'y'])
+x = None  # type: X
+a = x._asdict()
+
+[case testNamedTupleReplace]
+from collections import namedtuple
+
+X = namedtuple('X', ['x', 'y'])
+x = None  # type: X
+x = x._replace(y=5)
```

Comment:
True. I just don't know how to implement that, and thought that `AnyType` is better than nothing.